templates/vue/src/App.vue

```diff
@@ -57,6 +90,12 @@ export default {
   },
   methods: {
     ...mapMutations(["init"]),
+    refresh() {
+      this.$router.go()
+    },
+    redirectToLogin() {
+      window.location.href = `${window.location.origin}/wp-login.php?redirect_to=${window.location.pathname}`
```

Comment:
This should ideally include the `#` and everything after it too so that we can take the user back to exactly where they were in the given tapestry.